python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -287,6 +306,8 @@ def select_operator_to_run(
     cur_usage: TopologyResourceUsage,
     limits: ExecutionResources,
     ensure_at_least_one_running: bool,
+    execution_id: str,
+    autoscaling_state: Optional[AutoscalingState] = None,
```

Comment:
Make this a required arg to avoid errors.

Reply:
Done.